### PR TITLE
Added git clone feedback for user. Fixes #176

### LINKED ISSUE #176: Git remote clone never returns success
Sits there having returned zero, no success message given, no attempt to open/import project made.

```sh
~/git/Paladin> Paladin/Paladin -d -v 
Printing debug output with extra detail
Command: git: git --no-pager clone https://github.com/adamfowleruk/norel.git '/boot/home/Projects/norel' 2>&1
Cloning into '/boot/home/Projects/norel'...
git: out:
------------
Cloning into '/boot/home/Projects/norel'...
Source control command return value: 0
------------
```

Should give the user some indication of progress and completion, especially for larger repos.

### PR DESCRIPTION
**Pull request by adamfowleruk**

### CODE REVIEW

File: Paladin/SourceControl/SourceControl.cpp [hunk 1]
```diff
@@ -299,39 +308,67 @@ int
...
-	int32 pos = out.FindLast("Source control command return value");
+	//int32 pos = out.FindLast("Source control command return value");
+	
+	//int returnValue = atoi(out.String() + pos + strlen("Source control command return value: "));
+	//out.Truncate(pos);
+	out << "----------\n";
+	if (-1 == result) {
+		out << B_TRANSLATE("Command resulted in an error.\n");
+	} else {
+		out << 	B_TRANSLATE("Command succeeded. Use 'Import existing Project' "
```
Comment (humdingerb) [inline Paladin/SourceControl/SourceControl.cpp:363]: Shouldn't that be 'Import existing project'. Sentence-casing is used everywhere else.

Comment (adamfowleruk) [reply Paladin/SourceControl/SourceControl.cpp:363]: Yes very true. I missed that one. Feel free to fix if you have time. Otherwise, please lot as a bug.

Comment (humdingerb) [reply Paladin/SourceControl/SourceControl.cpp:363]: How about I fix that and newly generated en.catkeys which I upload at Polyglot?
Or do you expect major changes in the GUI strings? Additional or removed strings aren't a problem, only changed strings, as those have to be translated from scratch again.

Comment (adamfowleruk) [reply Paladin/SourceControl/SourceControl.cpp:363]: I don't think any will really change. I think there may still be some that don't yet exist. Feel free to upload new en.catkeys at polyglot and translate. Thanks!

**Pull request merged**